purchase_comment_template/views/report_purchaseorder.xml

```diff
@@ -0,0 +1,63 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <template id="report_purchaseorder_document_comments"
+      inherit_id="purchase.report_purchaseorder_document">
+      <xpath expr="//div[hasclass('oe_structure')][1]" position="before">
+        <template id="purchase_css" inherit_id="web.layout">
+            <xpath expr="//head" position="inside">
+                <link rel="stylesheet"
+                    href="/purchase_comment_template/static/src/css/purchase.css" />
+            </xpath>
+        </template>
+      </xpath>
+      <xpath expr="//table[hasclass('table-sm')]" position="before">
```

Comment:
There might be room for improvement for how this appears on the report, right now it stands on top.

Reply:
what do you propose?